src/sentry/web/frontend/shared_group_details.py

```diff
@@ -1,24 +1,40 @@
 from rest_framework.request import Request
 
+from sentry import analytics
 from sentry.models import Group
+from sentry.utils.assets import get_asset_url
+from sentry.utils.metatags import get_provider_by_user_agent
 from sentry.web.frontend.react_page import GenericReactPageView
 
 
 class SharedGroupDetailsView(GenericReactPageView):
     def meta_tags(self, request: Request, share_id):
+        obj = super().meta_tags(request)
         try:
             group = Group.objects.from_share_id(share_id)
         except Group.DoesNotExist:
             return {}
         if group.organization.flags.disable_shared_issues:
             return {}
+
+        user_agent = request.META.get("HTTP_USER_AGENT")
+        provider = None
+        if user_agent:
+            provider = get_provider_by_user_agent(user_agent)
+        if provider:
+            analytics.record("metalink.shared.scraped", provider=provider)
```

Comment:
@Neo-Zhixing I think it would be good if we can get more information than just the provider, like the issue ID. If we have that, we can join to the issues table and get a lot more info. And an organization ID would also be good to add.